store/event-store/src/test/java/org/calrissian/accumulorecipes/eventstore/impl/AccumuloEventStoreTest.java

```diff
@@ -136,6 +136,8 @@ public void testVisibility() {
         assertEquals(1, Iterables.get(actualEvent1, 1).size());
     }
 
+
+    @Ignore // ignoring for now. We may not need tuples to expire
```

Comment:
This feature was never officially released and was part of a previous optimization which was implemented mostly to fix issues that arose out of trying to do json marshalling. Though I don't mind linking the ticket- we shouldn't be spending too much time worrying about unreleased features.